docs/development.rst

```diff
@@ -0,0 +1,45 @@
+Building extension
+==================
+As with most VS Code extensions you need you need `Node.JS <https://nodejs.org/en/>`_ installed.
+
+The process if failry straightforward:
```

Comment:
`failry` -> `fairly`.

You'll need to add this to the toctree in `index.rst` as `development`. Also, it will be useful to move the `Developer Reference` section from `index.rst` to the top of this file.